missing `objtool` from kernel-devel

While troubleshooting an issue with an out-of-tree module, I found that it failed with an error like this when building in a Docker container:
> make[1]: *** No rule to make target 'kmod.o'

When building in a host container, the same invocation succeeded.

Docker command:
```
docker run -it --rm --net host \
  --mount type=bind,source=${HOME}/mod,target=/tmp/mod \
  --mount type=bind,source=${HOME}/bottlerocket/build/x86_64-bottlerocket-linux-gnu/sys-root/usr/lib/modules,target=/lib/modules \
  --mount type=bind,source=${HOME}/bottlerocket/build/x86_64-bottlerocket-linux-gnu/sys-root/usr/share/bottlerocket/squashfs-root,target=/usr/src/kernels \
  bottlerocket/sdk-x86_64:v0.9.0
```

Build command:
```
make -C /lib/modules/5.4.16/build/ M=`pwd` -d modules V=1
```

I compared the output of both builds. The failing build had references to `objtool`:
```
Trying rule prerequisite 'tools/objtool/objtool'.
Looking for a rule with intermediate file 'tools/objtool/objtool'.
Rejecting impossible rule prerequisite 'tools/objtool/objtool'.
```

Adding the following mount option to the Docker command worked around the problem:
```
docker run -it --rm --net host ... \
  --mount type=bind,source=/usr/bin/true,target=/usr/src/kernels/5.4.16/tools/objtool/objtool
  bottlerocket/sdk-x86_64:v0.9.0
```

This may mean we need to find a way to cross-compile `objtool` for our target and include it in the expected path. We might also need to put it in the SDK.

First it would be good to understand why this isn't needed on inside a host container.

--- a/packages/kernel/kernel.spec
+++ b/packages/kernel/kernel.spec
@@ -107,6 +107,11 @@ mkdir src_squashfs
 for file in $(cat kernel_devel_files); do
   install -D ${file} src_squashfs/%{version}/${file}
 done
+# if we have it, include objtool (not all arches support it yet)
+if [ "%{_cross_karch}" == "x86"  ]; then
+  install -D tools/objtool/objtool src_squashfs/%{version}/tools/objtool/objtool
+fi
+
 mksquashfs src_squashfs kernel-devel.squashfs
 install -D kernel-devel.squashfs %{buildroot}%{_cross_datadir}/bottlerocket/kernel-devel.squashfs
 install -d %{buildroot}%{kernel_sourcedir}

